### ISSUE TITLE
Add failure reason to tests that do not have it

### ISSUE THREAD

**Issue by driver733** [yegor256/cactoos#703]
Currently, the majority of tests do not have failure reasons. 
I believe that we should fix this by adding failure reasons to all tests that miss it.

**Comment by 0crat** [comment]
@yegor256[/z](https://www.0crat.com/u/yegor256) please, pay attention to this issue

**Comment by yegor256** [comment]
@driver733 we can't ensure this always to happen, that's why fixing it only once won't really help. Can we find a way to control it somehow in the build?

**Comment by driver733** [comment]
@yegor256 https://github.com/yegor256/cactoos/issues/704#issuecomment-367378441

**Comment by yegor256** [comment]
@driver733 it's a duplicate of #704
